B827EBFFFEE95635.json

```diff
@@ -0,0 +1,35 @@
+{
+
+"gateway_conf": {
+
+"gateway_ID": "B827EBFFFEE95635",
+
+"servers": [
+
+{
+
+"server_address": "router.eu.thethings.network",
+
+"serv_port_up": 1700,
+
+"serv_port_down": 1700,
+
+"serv_enabled": true
+
+}
+
+],
+
+"ref_latitude": 40.971982,
+
+"ref_longitude": -5.652831,
+
+"ref_altitude": 810,
+
+"contact_email": “jc@alfaiot.com”,
```

Comment:
Please use the normal double quotes (`"`), these ones are invalid in a json file.